pkg/restore/import.go

```diff
@@ -304,26 +340,13 @@ func (importer *FileImporter) ingestSST(
 		Context: reqCtx,
 		Sst:     sstMeta,
 	}
-	log.Debug("download SST", zap.Stringer("sstMeta", sstMeta))
+	log.Debug("ingest SST", zap.Stringer("sstMeta", sstMeta), zap.Reflect("leader", leader))
 	resp, err := importer.importClient.IngestSST(importer.ctx, leader.GetStoreId(), req)
 	if err != nil {
-		if strings.Contains(err.Error(), "RegionNotFound") {
-			return errors.Trace(errRegionNotFound)
-		}
-		return errors.Trace(err)
-	}
-	respErr := resp.GetError()
-	if respErr != nil {
-		log.Debug("ingest sst resp error", zap.Stringer("error", respErr))
-		if respErr.GetKeyNotInRegion() != nil {
-			return errors.Trace(errKeyNotInRegion)
-		}
-		if respErr.GetNotLeader() != nil {
-			return errors.Trace(errNotLeader)
-		}
-		return errors.Wrap(errResp, respErr.String())
+		log.Error("ingest sst error", zap.Error(err))
```

Comment:
If you log the error here there will be two error logs (another is "ingest file failed"). Maybe redundant.